### DISCUSSION TITLE
How to use PropertyEquals selector with non-string value?

### DISCUSSION THREAD

**Discussion by LJN-hzleaper** [AvaloniaUI/Avalonia#10642] [Q&A]
For example, I have an enum:
```
public enum MyEnum
{
    None, Hello, Bye
}
```
and an Image:
```
<Image Tag="{Binding E}" Width="64" Height="64"></Image>
```
where `E` is a MyEnum in DataContext.
I want to use style to change the image Source according to its Tag. I have tried this code, but it doesn't work.
```
<Style Selector="Image[Tag=Bye]">
    <Setter Property="Source" Value="{StaticResource ImageAdd}"></Setter>
</Style>
```

**Comment by timunie** [comment]
I'm not sure if this is supported. But as a workaround I'd just use a converter to select the source.
